bigtable/test_bigtable.py

```diff
@@ -10,12 +10,16 @@
     See: https://cloud.google.com/bigtable/docs/cbt-overview
 """
 import os
+import sys
 import unittest
 
+from absl import flags
 from parameterized import parameterized
-
 from integration_tests.dataproc_test_case import DataprocTestCase
 
+FLAGS = flags.FLAGS
```

Comment:
Can we move flag initialization in the super class `DataprocTestCase` so it will be shared between all the tests?

Reply:
Now that there is only the --image_version flag which is the same across all the tests this makes a lot of sense. Good catch.